src/configure-balena-root-ca.sh

```diff
@@ -3,6 +3,10 @@ if test -n "${BALENA_ROOT_CA}"; then
     echo "Installing self-signed balena root CA..."
     echo "${BALENA_ROOT_CA}" | base64 -d > /usr/local/share/ca-certificates/balenaRootCA.crt
     update-ca-certificates
+elif test -n "${ROOT_CA}"; then
```

Comment:
I think you can use
```bash
ROOT_CA=${ROOT_CA:-$BALENA_ROOT_CA}
```
at the top to avoid the duplicate `if`s

Reply:
@Page- great idea